nf_core/lint/files_unchanged.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+
+import logging
+import filecmp
+import os
+import tempfile
+
+import nf_core.create
+
+
+def files_unchanged(self):
+    """Checks that certain pipeline files are not modified from template output.
+
+    Iterates through the pipeline's directory content and compares specified files
+    against output from the template using the pipeline's metadata. File content
+    should not be modified / missing.
+
+    Files that must be unchanged::
+
+        'CODE_OF_CONDUCT.md',
+        ['LICENSE', 'LICENSE.md', 'LICENCE', 'LICENCE.md'], # NB: British / American spelling
+        '.gitattributes',
+        '.github/.dockstore.yml',
+        '.github/CONTRIBUTING.md',
+        '.github/PULL_REQUEST_TEMPLATE.md',
+        '.github/markdownlint.yml',
+        '.github/ISSUE_TEMPLATE/bug_report.md',
+        '.github/ISSUE_TEMPLATE/config.yml',
+        '.github/ISSUE_TEMPLATE/feature_request.md',
+        '.github/workflows/awsfulltest.yml',
```

Comment:
I agree but then the `TODO` string should be removed from here on the template, and `-profile test` changed to `-profile test_full`

Reply:
I guess that the only down side of this is if we want people to be able to run _multiple_ AWS full size tests (eg. the `rnaseq` pipeline with different aligners).

Though actually if we move this to the `files_partial` list instead then it should be fine as people could add additional workflow steps below it.. Ok I'll play with the workflow files a little then :+1:

Reply:
Yeah, same as you say below in https://github.com/nf-core/tools/pull/846/files#r573026542 I realise..

Problem with this is that people can't edit the `strategy` `matrix`.

I wonder if we're better ditching these files from this test but rewriting the existing tests that we have for them to be a bit simpler - just checking for equivalence in the YAML for the majority of keys.

Reply:
Removed from this test 👍🏻

Reply:
ok yeah that sounds good